pep-0696.rst

```diff
@@ -184,16 +187,93 @@ default to the type of ``start`` and step default to ``int | None``.
    StopT = TypeVar("StopT", default=StartT)
    StepT = TypeVar("StepT", default=int | None)
 
-   class slice(Generic[StartT, StopT, StepT]): ...  # Valid
+   class slice(Generic[StartT, StopT, StepT]): ...
+
+   reveal_type(slice())                        # type is slice[int, int, int | None]
+   reveal_type(slice[str]())                   # type is slice[str, str, int | None]
+   reveal_type(slice[str, bool, timedelta]())  # type is slice[str, bool, timedelta]
+
+When using a ``TypeVarLike`` as the default to another ``TypeVarLike``.
+Where ``T1`` is the default for ``T2`` the following rules apply.
+
+Scoping Rules
+~~~~~~~~~~~~~
+
+``T1`` must be used before ``T2`` in the parameter list of the generic
+or be bound in an outer class or function scope.
+
+.. code:: py
+
+   DefaultT = TypeVar("DefaultT", default=T)
 
-   reveal_type(slice())                       # type is slice[int, int, int | None]
-   reveal_type(slice[str]())                  # type is slice[str, str, int | None]
-   reveal_type(slice[str, str, timedelta]())  # type is slice[str, str, timedelta]
+   class Foo(Generic[T, DefaultT]): ...   # Valid
+   def bar(x: T, y: DefaultT): ...        # Valid
+   class Foo(Generic[T]):
+       class Bar(Generic[DefaultT]): ...  # Valid
+   def outer(x: T):
+       def inner(y: DefaultT): ...        # Valid
 
-   StartT = TypeVar("StartT", default="StopT")
+   StartT = TypeVar("StartT", default="StopT")  # Swapped defaults around from previous example
    StopT = TypeVar("StopT", default=int)
    class slice(Generic[StartT, StopT, StepT]): ...
-                       ^^^^^^  # Invalid: ordering does not allow StopT to bound yet
+                     # ^^^^^^ Invalid: ordering does not allow StopT to be bound
+   def baz(x: DefaultT, y: T): ...
+            # ^^^^^^^^ Invalid: ordering does not allow DefaultT to be bound
+
+Bound Rules
+~~~~~~~~~~~
+
+``T2``'s bound must be a subtype of ``T1``'s bound.
+
+.. code:: py
+
+   T = TypeVar("T", bound=float)
+   TypeVar("Ok", default=T, bound=int)        # Valid
+   TypeVar("AlsoOk", default=T, bound=float)  # Valid
+   TypeVar("Invalid", default=T, bound=str)   # Invalid: str is not a subtype of float
+
+Constraint Rules
+~~~~~~~~~~~~~~~~
+
+``T2``'s constraint must either contain the bound of ``T1`` or the
```

Comment:
```suggestion
Either ``T2``'s constraint must contain the bound of ``T1``, or the
```

A comma here helps disambiguate the intended grouping (assuming I've parsed the intent correctly as "(T2's constraint must contain the bound of T2) or (the constrains of T2 must be a superset of the constraints of T2)"